clam/clamservice.py

```diff
@@ -993,6 +1006,7 @@ def abort(project, user):
         while not os.path.exists(Project.path(project, user) + ".done"):
             printdebug("Waiting for process to die")
             time.sleep(1)
+        Project.updateindex(user, project, 0.0, clam.common.status.READY)
```

Comment:
This may be a bit confusing, but when a project is done the status is not `READY` but `DONE`. Status `READY` means the project is created and ready for user input and to be run.

Actually, the index updating should not be needed at this point, because the dispatcher already handled it after a process ends. See https://github.com/proycon/clam/blob/master/clam/clamdispatcher.py#L38 and https://github.com/proycon/clam/blob/master/clam/clamdispatcher.py#L258